poetry.lock

```diff
@@ -686,6 +686,14 @@ python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
 [package.dependencies]
 six = "*"
 
+[[package]]
+name = "rfc3986-validator"
+version = "0.1.1"
+description = "Pure python rfc3986 validator"
+category = "main"
+optional = false
+python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
+
 [[package]]
 name = "rfc3987"
```

Comment:
Something seems a bit off to me here - shouldn't this one be removed after the change in dependencies, since it's one of the GPL packages? Might try doing a `poetry remove jsonschema` and `poetry add jsonschema --extras format-nongpl` and see if that changes things at all.

Reply:
Hmmm. I ran those commands to remove jsonschema from the poetry env and re-install it with the format-nongpl. The command to remove did remove the rfc3986-validator package

```bash
(schema-enforcer-tr67_61X-py3.9) ➜  schema-enforcer git:(issue-133) poetry remove jsonschema
Updating dependencies
Resolving dependencies... (0.3s)

Writing lock file

Package operations: 0 installs, 0 updates, 9 removals

  • Removing arrow (1.2.2)
  • Removing fqdn (1.5.1)
  • Removing isoduration (20.11.0)
  • Removing jsonschema (4.7.1)
  • Removing pyrsistent (0.18.1)
  • Removing python-dateutil (2.8.2)
  • Removing rfc3986-validator (0.1.1)
  • Removing uri-template (1.2.0)
  • Removing webcolors (1.12)
```

The command to add with non-gpl added it back in.

```bash
(schema-enforcer-tr67_61X-py3.9) ➜  schema-enforcer git:(issue-133) ✗ poetry add jsonschema --extras format-nongpl                                                     
Using version ^4.7.1 for jsonschema

Updating dependencies
Resolving dependencies... (0.5s)

Writing lock file

Package operations: 9 installs, 0 updates, 0 removals

  • Installing python-dateutil (2.8.2)
  • Installing arrow (1.2.2)
  • Installing fqdn (1.5.1)
  • Installing isoduration (20.11.0)
  • Installing pyrsistent (0.18.1)
  • Installing rfc3986-validator (0.1.1)
  • Installing uri-template (1.2.0)
  • Installing webcolors (1.12)
  • Installing jsonschema (4.7.1)
```